tools/cockpit.spec.in

```diff
@@ -90,6 +90,10 @@ BuildRequires: autoconf automake
 BuildRequires: make
 BuildRequires: /usr/bin/python3
 BuildRequires: gettext >= 0.19.7
+# RHEL 8's gettext does not yet have metainfo.its
+%if 0%{?rhel} && 0%{?rhel} <= 8
+BuildRequires: libappstream-glib-devel
```

Comment:
This feels deeply strange.  If the version of gettext (listed above) doesn't have the required stuff, then we should unconditionally build-depend this.  Alternatively, do an `%else` here with different gettext versions.

Reply:
It's a [direct port from starter-kit](https://github.com/cockpit-project/starter-kit/commit/4377c4fe19ebe373da2a8d77fa2d75f12ca3dc17#diff-0b0aaa8dc745dda87fea8315e513b3d259840c07f9d5a2d5559766e20d38c2bbR13), but yes, this could be a bit cleaner. Is this ok for you?
```spec
%if 0%{?rhel} && 0%{?rhel} <= 8
# RHEL 8's gettext does not yet have metainfo.its
BuildRequires: gettext >= 0.19.7
BuildRequires: libappstream-glib-devel
%else
BuildRequires: gettext >= 0.21
%endif
```

Reply:
Seems OK.  I don't think it really matters so much to have an extra build depend that we don't need.  I do like that this will be more obviously removable when we drop RHEL 8, though :)

Reply:
Fixed, thanks!